packages/ui/src/proposals/components/StatisticsPreview/ProposalStatistics.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import { StatisticBar, Statistics, TwoRowStatistic } from '@/common/components/statistics'
+import { VoteCount } from '@/proposals/hooks/useVoteCount'
+import { ProposalConstants } from '@/proposals/types'
+
+const tooltipLinkURL = 'https://joystream.gitbook.io/joystream-handbook/governance/proposals#concepts'
+
+interface ProposalStatisticsProps {
+  voteCount: VoteCount
+  constants: ProposalConstants
+}
+export const ProposalStatistics = ({ voteCount, constants }: ProposalStatisticsProps) => {
+  const { approve, slash, total, remain } = voteCount
+  const councilSize = total + remain
+  const {
+    approvalQuorumPercentage,
+    approvalThresholdPercentage,
+    slashingQuorumPercentage,
+    slashingThresholdPercentage,
+  } = constants
+
+  const approvalQuorumRatio = approvalQuorumPercentage / 100
+  const approvalThresholdRatio = approvalThresholdPercentage / 100
+  const slashingQuorumRatio = slashingQuorumPercentage / 100
+  const slashingThresholdRatio = slashingThresholdPercentage / 100
+
+  // NOTE: use ceil since these are minimums to reach
+  const approvalQuorum = Math.ceil(councilSize * approvalQuorumRatio)
+  const slashingQuorum = Math.ceil(councilSize * slashingQuorumRatio)
+
+  const quorumRatio = councilSize && total / councilSize
+  const approvalRatio = total && approve / total
+  const slashingRatio = total && slash / total
+
+  const approvalPercentOfTheThreshold = approvalThresholdRatio
+    ? Math.floor((approvalRatio / approvalThresholdRatio) * 100)
+    : '∞'
+  const slashingPercentOfTheThreshold = slashingThresholdRatio
+    ? Math.floor((slashingRatio / slashingThresholdRatio) * 100)
+    : '∞'
```

Comment:
You're right I got confused by the design: ![](https://user-images.githubusercontent.com/6571453/123389728-76d0b000-d59a-11eb-86ad-2f03b6b130fa.png), because it looks like it's 50% approval out of the 51% threshold, but this would have been very confusing :sweat: ! Also I just realized the bars on the wiredframe match the percentages correctly.